.github/scripts/validate_scale_config.py

```diff
@@ -31,6 +34,30 @@
 PREFIX_LF = "lf."
 PREFIX_LF_CANARY = "lf.c."
 
+_RUNNER_BASE_JSCHEMA = {
+    "type": "object",
+    "additionalProperties": False,
+    "properties": {
+        "ami_experiment": {"type": "object"},
+        "ami": {"type": "string"},
+        "disk_size": {"type": "number"},
+        "instance_type": {"type": "string"},
+        "is_ephemeral": {"type": "boolean"},
+        "labels": {"type": "array", "items": {"type": "string"}},
```

Comment:
I don't quite understand what labels is used for, esp since it's not actually used on our scale config.

Reply:
IIRC it is a feature request by either Andrey or Nikita a while back ago for some migration / better engineering, I believe related to windows runners AMI migrations or something, but I never seen it being used.